src/test/java/technology/semi/weaviate/integration/client/graphql/ClientGraphQLTest.java

```diff
@@ -779,4 +827,42 @@ private String getIdFromResponse(GraphQLResponse resp) {
     String id = (String) additional.get("id");
     return id;
   }
+
+  private String getCreationTimeUnixFromResponse(GraphQLResponse resp) {
+    assertNotNull(resp);
+    assertNull(resp.getErrors());
+    assertNotNull(resp.getData());
+    assertTrue(resp.getData() instanceof Map);
+    Map data = (Map) resp.getData();
+    assertNotNull(data.get("Get"));
+    assertTrue(data.get("Get") instanceof Map);
+    Map get = (Map) data.get("Get");
+    assertNotNull(get.get("Pizza"));
+    assertTrue(get.get("Pizza") instanceof List);
+    List pizza = (List) get.get("Pizza");
+    assertTrue(pizza.get(0) instanceof Map);
+    Map firstPizza = (Map) pizza.get(0);
+    Map additional = (Map) firstPizza.get("_additional");
+    String time = (String) additional.get("creationTimeUnix");
+    return time;
+  }
+
+  private String getLastUpdateTimeUnixFromResponse(GraphQLResponse resp) {
+    assertNotNull(resp);
+    assertNull(resp.getErrors());
+    assertNotNull(resp.getData());
+    assertTrue(resp.getData() instanceof Map);
+    Map data = (Map) resp.getData();
+    assertNotNull(data.get("Get"));
+    assertTrue(data.get("Get") instanceof Map);
+    Map get = (Map) data.get("Get");
+    assertNotNull(get.get("Pizza"));
+    assertTrue(get.get("Pizza") instanceof List);
+    List pizza = (List) get.get("Pizza");
+    assertTrue(pizza.get(0) instanceof Map);
+    Map firstPizza = (Map) pizza.get(0);
+    Map additional = (Map) firstPizza.get("_additional");
+    String time = (String) additional.get("lastUpdateTimeUnix");
+    return time;
+  }
```

Comment:
since those methods are almost identical, maybe 3rd one should be created, accepting field name as argument, so those two could call new one with field name.

Reply:
you're right, this should work for all `_additional` fields, so maybe i will do 
```
getAdditionalFieldFromResposne(String fieldName)
```

Reply:
fixed accordingly